example/src/main/deploy/kubernetes/kubernetes_deploy.sh

```diff
@@ -85,6 +85,7 @@ kubectl create secret generic hono-service-device-registry-conf \
   --from-file=$CERTS/auth-server-cert.pem \
   --from-file=$CERTS/trusted-certs.pem \
   --from-file=$CONFIG/example-credentials.json \
+  --from-file=$CONFIG/example-tenants.json \
```

Comment:
do we also need to add this to `openshift_deploy.sh`?

Reply:
:+1: Thanks, will add this!